@xen-orchestra/web-core/lib/components/ui/table-pagination/UiTablePagination.vue

```diff
@@ -0,0 +1,212 @@
+<!-- v2 -->
+<template>
+  <div class="ui-table-pagination">
+    <span class="typo p3-regular label">
+      {{ $t('core.select.n-object-of', { from: startIndex, to: endIndex, total: totalItems }) }}
+    </span>
+
+    <div class="buttons-content">
+      <UiButtonIcon
+        :disabled="isFirstPage"
+        accent="info"
+        class="icon"
+        size="small"
+        :icon="faAngleDoubleLeft"
+        @click="goToFirstPage"
```

Comment:
This would not change anything here, but as a "guideline" we try to use this syntax: `goToFirstPage()` to avoid some unwanted behavior in some cases.
```suggestion
        @click="goToFirstPage()"
```
Same applies for the other usages in the template.

Reply:
yes , it's an oversight

Reply:
A quick reminder for those wondering why we consider this to be bad practice:

`@click="deleteMe"` is the equivalent of `@click="deleteMe($event)"` (or `@click="(event) => deleteMe(event)"`)

Let's take this code:

```vue
<template>
  <button @click="deleteMe">Delete</button>
  <button @click="deleteMe(true)">Force Delete</button>
</template>

<script>
function deleteMe(force?: boolean) {
  if(force) { ... }
  else { ... }
}
</script>
```

In this example, the first button will call `deleteMe(event)` and thus will always take the first condition path.